server/middleware/cron/events.js

```diff
@@ -1,13 +1,26 @@
 import cron from 'cron'
 import puppeteer from 'puppeteer'
 
+import Festival from '../../database/models/festival.js'
+
 // Job starts when clock hits the 0 hour, the 0 minute
 const job = cron.job('0 0 * * *', () => {
-  getFestivalData()
+  setFestivalData()
+})
+
+Festival.findOne().then(latestFestival => {
+  const latestFestivalUpdate = Number(latestFestival.timestamp)
+  const day = 1000 * 60 * 60 * 24
+  const isRecentlyUpdated = Date.now() - day < latestFestivalUpdate
+
+  if (!isRecentlyUpdated) {
+    setFestivalData()
+  }
 })
+
 export default job.start()
 
-async function getFestivalData() {
+async function setFestivalData() {
```

Comment:
Really just a minor detail, but I see you're using normal functions as well as arrow functions. Might want to use one or the other consistently 😊. Everything else looks good!

Reply:
Changed it! Thanks!